### PR TITLE
Notification summary

### PR DESCRIPTION
**Pull request by hdonnay**
This adds a toggle and wires it through to the config object.

### CODE REVIEW

**Review by Allda — DISMISSED** [review]
LGTM

File: notifier/processor_create_test.go [hunk 1]
```diff
@@ -2,260 +2,331 @@ package notifier
...
+	// Optional arguments.
+	NoSummary   bool
+	Err         bool
+	PrevUUID    uuid.UUID
+	UpdaterName string
+	OpRef       uuid.UUID
+	EventDate   time.Time
+	// These functions are like the Named methods on various interfaces, with
+	// the addition of a testing.T object for convenience.
+	UpdateDiff        func(*testing.T, context.Context, uuid.UUID, uuid.UUID) (*driver.UpdateDiff, error)
```
Comment (ldelossa) [inline notifier/processor_create_test.go]: this is conflating the use of a mock, and a test case. I think its confusing to have them. this seems like a use of "clever" over "obvious"

**Review by ldelossa — DISMISSED** [review]
I'm not a fan of the complexity added to the tests. I understand some of the reasoning, but keeping it flat, dumb, and obvious, since its a test, was a particular goal of mine in the notifier code. 

The added complexity doesn't buy much, and confuses the usage of mocks.

File: notifier/summary_test.go [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+	e := Event{
+		updater: updater,
+		uo: driver.UpdateOperation{
+			Ref:     uuid.New(),
+			Updater: updater,
+		},
+	}
+	manifest := "sha256:8d502da610b3c153d5aedaaf5323c0d49f61401d4791b4b1ffe9e36c6cbe09a0"
+	vs := []claircore.Vulnerability{
+		{ID: "🖳", Name: "uncool vulnerability"},
```
Comment (ldelossa) [inline notifier/summary_test.go:32]: lol why the emojis ?

**Review by ldelossa — APPROVED** [review]

File: notifier/summary_test.go (hunk 2 above)
Comment (hdonnay) [reply notifier/summary_test.go:32]: Everyone loves emojis 😘🆒🆒🆒🤪

**Pull request merged**